pkg/backend/httpstate/client/client.go

```diff
@@ -305,6 +305,46 @@ type serviceTokenInfo struct {
 	Team         string `json:"team,omitempty"`
 }
 
+func (pc *Client) ExchangeOidcToken(
+	oidcToken string, org string, scope string, expiration int,
+) (*apitype.TokenExchangeGrantResponse, error) {
+	//nolint:gosec // These are token type URNs, not credentials
+	requestedTokenType := "urn:pulumi:token-type:access_token:organization"
+	if strings.HasPrefix(scope, "team:") {
+		requestedTokenType = "urn:pulumi:token-type:access_token:team" //nolint:gosec // token type URN
+	}
+	if strings.HasPrefix(scope, "user:") {
+		requestedTokenType = "urn:pulumi:token-type:access_token:personal" //nolint:gosec // token type URN
+	}
```

Comment:
Super nit: instead of disabling the lints here, put these in module consts to reduce clutter in this bit of logic which seems very critical?